robottelo/ui/contentviews.py

```diff
@@ -55,18 +55,13 @@ def create(self, name, label=None, description=None, is_composite=False):
         self.wait_for_ajax()
         self.click(common_locators['create'])
 
-    def delete(self, name, really=True):
+    def delete(self, name):
```

Comment:
Either approach works for me. My only concern is to have an issue to track the future improvement.